src/components/SelectBox.js

```diff
@@ -43,16 +44,30 @@ class SelectBox extends React.Component {
       this.props.onChange(id)
     }
   }
+  getValidationState () {
+    const { validationState } = this.props
+    switch (validationState) {
+      case 'error':
+        return style['selectBox-has-error']
+      case 'warning':
+        return style['selectBox-has-warning']
```

Comment:
I still think we should avoid adding unnecessary code. We have no any use for this right now and we don't know if we will in the future.

Reply:
@sgratch @sjd78 @rszwajko ,
@hstastna and I have disagreements about if we need or don't need the functionality which are not used right now but we creating the same interface as PF3 [`FormControl`](https://5c507d49471426000887a6a7--react-bootstrap.netlify.app/components/forms/#forms-validation) component
WDYT guys?

Reply:
> we should avoid adding unnecessary code

> we creating the same interface as PF3 FormControl component

Both views make sense. It's up to you @bamsalem

Reply:
I thought about that, and we don't want to maintain unnecessary code so I'll remove that code :)